how-to/how-to-add-telerik-ui-for-nativescript-package.md

```diff
@@ -0,0 +1,69 @@
+---
+title: How to Include the {{site.tuin}} Package
+description: A tutorial for adding {{site.tuin}} to a mobile application in AppBuilder. You can use the UI controls by including the free limited edition or by downloading the pro version package from your account and importing it in your app
+type: how-to
+page_title: How to Add {{site.tuin}} in AppBuilder
+slug: include-nativescript-ui-package
+position:
+tags: appbuilder, nativescript
+teampulseid:
+ticketid:
+pitsid:
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Package management</td>
+  </tr>
+  <tr>
+    <td>Mobile Development type</td>
+    <td>Native (NativeScript)</td>
+  </tr>
+</table>
+
+## Description
+[{{site.tuin}}](http://www.telerik.com/nativescript-ui) allows you to incorporate native UI controls in your NativeScript applications. You can choose between a free limited edition or a complete edition available with specific subscription plan. To have access to the UI components, you need to add the {{site.tuin}} package to your app in AppBuilder.
+
+## Solution
+### {{site.tuin}} (Free)
+1. Open your app in AppBuilder.
+1. In the **Project Navigator** (**Solution Explorer** in Microsoft Visual Studio), right-click the second node from the top and select **Manage Packages**.
+1. On the **NPM** tab, search for `nativescript-telerik-ui` and select the package.
+1. (Optional) Select a package version.
+1. Click **Install**.
+
+> **IMPORTANT:** The NPM `nativescript-telerik-ui-pro` package is for the Trial version.
+
+<br />
+### {{site.tuin}} (Pro)
+1. Navigate to [www.telerik.com/account]() and log in your account.
```

Comment:
The syntax looks pretty weird to me. Is this working when built or is it rendered as an empty link?